e2e/test/scenarios/embedding/sdk-iframe-embedding/sdk-iframe-embedding.cy.spec.ts

```diff
@@ -266,6 +310,104 @@ describe("scenarios > embedding > sdk iframe embedding", () => {
   });
 });
 
+H.describeWithSnowplowEE.only(
```

Comment:
I think we need to remove the focused test `.only` here.

Reply:
fudge. How that hell that get in here.